scripts/parse-blacklisted-modules.js

```diff
@@ -5,7 +5,9 @@ const network = process.argv[2]
 const blacklistedModules = require('../blacklisted-modules')[network]
 
 console.log(`Blacklisting modules: ${blacklistedModules}`)
+const outputPath = `${process.cwd()}/helpers/blacklisted-modules.ts`
+if (fs.existsSync(outputPath)) fs.unlinkSync(outputPath)
 
-const parsedModules = `export const BLACKLISTED_MODULES: string[] = [${blacklistedModules.map(x => `"${x}"`).join(', ')}]`
-
-fs.writeFileSync(`${process.cwd()}/helpers/blacklisted-modules.ts`, parsedModules)
+const header = `// This file was autogenerated at deployment time picking the list of blacklisted modules defined in "blacklisted-modules.js" for network ${network}`
+const modulesExport = `export const BLACKLISTED_MODULES: string[] = [${blacklistedModules.map(x => `"${x}"`).join(', ')}]`
+fs.writeFileSync(outputPath, `${header}\n\n${modulesExport}\n`)
```

Comment:
```suggestion
const file = `// This file was autogenerated at deployment time picking the list of blacklisted modules defined in "blacklisted-modules.js" for network ${network}`


export const BLACKLISTED_MODULES: string[] = [${blacklistedModules.map(x => `"${x}"`).join(', ')}]
`
fs.writeFileSync(outputPath, `${file}`)
```

Perhaps a multiline string is nicer :)